vue/components/ui/molecules/table/table.vue

```diff
@@ -275,6 +283,9 @@ export const Table = {
             const order = this.reverseData ? "ascending" : "descending";
             return this.sortData === column ? `active ${order}` : "";
         },
+        columnStyle(column) {
+            return { width: column.width, height: `${this.headerHeight}px` };

```

Comment:
We are using px for sizes. Also, make it optional: 
`            const base = {};
            if (this.width) base.width = `${this.width}px`;
            if (this.headerHeight) base.height = `${this.headerHeight}px`;
            return base;`